nameserver/http.go

```diff
@@ -9,18 +8,9 @@ import (
 	"log"
 	"net"
 	"net/http"
-	"strings"
+	"github.com/gorilla/mux"
```

Comment:
Please run 'go fmt' on code before committing (which would move this line earlier, in alphabetical order).